### PR TITLE
gpu: Make option compares case insensitive

### PR DESCRIPTION
**Pull request by Tony-LunarG**
Vkconfig uses caps for TRUE or FALSE, so make option compares case insensitive

### CODE REVIEW

**Comment by mark-lunarg** [comment]
It'd also be good (sometime) to make the option identifiers themselves case-insensitive and to generalize the separator handling so that it would accept commas, semicolons, colons, etc.

File: layers/debug_printf.cpp [hunk 1]
```diff
@@ -60,10 +60,14 @@ void DebugPrintf::PostCallRecordCreateDevice(VkPhysicalDevice physicalDevice, co
 
     const char *size_string = getLayerOption("khronos_validation.printf_buffer_size");
     device_debug_printf->output_buffer_size = *size_string ? atoi(size_string) : 1024;
-    const char *verbose_string = getLayerOption("khronos_validation.printf_verbose");
-    device_debug_printf->verbose = *verbose_string ? !strcmp(verbose_string, "true") : false;
-    const char *stdout_string = getLayerOption("khronos_validation.printf_to_stdout");
-    device_debug_printf->use_stdout = *stdout_string ? !strcmp(stdout_string, "true") : false;
+
+    std::string verbose_string = getLayerOption("khronos_validation.printf_verbose");
+    transform(verbose_string.begin(), verbose_string.end(), verbose_string.begin(), ::tolower);
```
Comment (ncesario-lunarg) [inline layers/debug_printf.cpp:65]: Probably equivalent, but any particular reason to use the C version of `tolower` vs `std::tolower`?

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

File: layers/debug_printf.cpp (hunk 1 above)
Comment (Tony-LunarG) [reply layers/debug_printf.cpp:65]: No.  Do you prefer std::tolower?  Easy enough to change

Comment (ncesario-lunarg) [reply layers/debug_printf.cpp:65]: Nope, just curious if there was a any difference. This change LGTM!

Comment (christophe-lunarg) [reply layers/debug_printf.cpp:65]: Erm, maybe vkconfig should output lower case then...

Comment (christophe-lunarg) [reply layers/debug_printf.cpp:65]: Others layers convert to tolower, so probably this change is right to be more consistent with the rest of the layer ecossytem.

**Pull request merged**